lib/util/problem.js

```diff
@@ -193,6 +193,8 @@ const problems = {
     // returned in case no external ODK Analytics service is configured.
     analyticsNotConfigured: problem(501.9, () => 'This ODK Central has not been configured to report Analytics. Please contact your server administrator.'),
 
+    invalidParameters: problem(501.10, (details) => `Invalid parameters passed. ${details}`),
```

Comment:
Because of some string/number issue (I forget where), we skip `.10` errors and go straight to `.11` errors so they don't wind up getting mixed up with `.1` errors  (if we wind up keeping this error)